src/databases/lowdb.js

```diff
@@ -9,7 +9,7 @@ const db = low(adapter);
 
 db.defaults({processedEvents: []}).write();
 
-export function saveParsedEvent(event: Event): Promise<any> {
+export function saveParsedEvent(event: Event): Promise<boolean> {
```

Comment:
The new implementation without `any` is far better :)
I can see just one small problema here, which is that flow does not know for sure that the return type is correct. In fact, the return code `db.get(...).push(...).write()` is not typed and probably should be.
The return type could be fine as `Promise<boolean>`, or could probably be `Promise<void>`, in case it fits better with the db call method (especially `write()`). I guess that trying to add a type to that will show what the best solution is.